todo_app/app.py

```diff
@@ -1,23 +1,66 @@
-from flask import Flask, request, redirect, render_template
+from flask import Flask, request, redirect, render_template, url_for
 
 from todo_app.flask_config import Config
-from todo_app.data.session_items import get_items, add_item
+from todo_app.data.trello_items import get_items, add_item, delete_item, completed_item, not_started_item
+
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    items = get_items()
-    #print(items)
+    items = get_items(app.config["BOARD_ID"],app.config["API_KEY"],app.config["API_TOKEN"])
     return render_template('index.html', items=items)
     
 @app.route('/additem', methods=['POST'])
 def addnewtodo():
-    newtodo= request.form.get('nitem')
-    if newtodo != 'Enter ToDo':
-        add_item(newtodo)
+    newtodo= request.form.get('new_item')
+    if newtodo != '':
+        add_item(newtodo,app.config["BOARD_ID"],app.config["API_KEY"],app.config["API_TOKEN"])
+    return redirect('/')
+
+@app.route('/cards/<int:id>/completed', methods=['GET','POST'])
```

Comment:
Is there a reason you're supporting both GET and POST? It's likely that GET doesn't work (because if you make a GET request with a form then it doesn't actually pass it through as a form, it passes the info as query parameters!) so we're potentially making a promise we can't keep. Restricting things to the used/tested route is often a better approach than trying to allow potentially untested approaches